python/doc/theory/data_analysis/qqplot_graph.rst

```diff
@@ -1,73 +1,54 @@
 .. _qqplot_graph:
 
-Using QQ-plot to compare two samples
-------------------------------------
-
-Let :math:`X` be a scalar uncertain variable modeled as a random
-variable. This method deals with the construction of a dataset prior to
-the choice of a probability distribution for :math:`X`. A QQ-plot (where
-“QQ” stands for “quantile-quantile”) is a tool that may be used to
-compare two samples :math:`\left\{x_1,\ldots,x_N \right\}` and
-:math:`\left\{x'_1,\ldots,x'_M \right\}`; the goal is to determine
-graphically whether these two samples come from the same probability
-distribution or not. If this is the case, the two samples should be
-aggregated in order to increase the robustness of further statistical
-analysis.
-
-A QQ-plot is based on the notion of quantile. The
-:math:`\alpha`-quantile :math:`q_{X}(\alpha)` of :math:`X`, where
-:math:`\alpha \in (0, 1)`, is defined as follows:
+QQ-plot
+-------
+
+The Quantile - Quantile - Plot (QQ Plot) allows one to validate whether
```

Comment:
The verb "allow" is not appropriate, because there is no need to use some particular law or ask to any judge to use this method: no policeman will put us under arrest here if we do not use that method (see [^EnableVsAllow]).
```suggestion
The Quantile - Quantile - Plot (QQ Plot) enables to validate whether
```

[^EnableVsAllow]: https://letlucyedit.wordpress.com/2013/04/03/enable-or-allow-why-you-need-to-get-it-right/